libnmstate/nm/applier.py

```diff
@@ -132,6 +134,10 @@ def set_ifaces_admin_state(ifaces_desired_state, con_profiles=()):
         nmdev = device.get_device_by_name(ifname)
         if not nmdev:
             if ifname in new_ifaces and iface_desired_state['state'] == 'up':
+                if _is_master_iface(
+                    iface_desired_state
+                ) and not iface_desired_state.get(MASTER_METADATA):
+                    new_master_not_enslaved_ifaces.add(ifname)
                 if iface_desired_state['type'] == ovs.INTERNAL_INTERFACE_TYPE:
```

Comment:
It will be safer to change this to an `elif`.

Reply:
Done.